crates/common/src/config/signer.rs

```diff
@@ -106,6 +124,11 @@ fn default_signer_image() -> String {
     SIGNER_IMAGE_DEFAULT.to_string()
 }
 
+fn default_tls_mode() -> TlsMode {
+    TlsMode::Insecure // To make the default use TLS, do
+                      // TlsMode::Certificate(PathBuf::from("./certs"))
```

Comment:
Are these comments meant to stay here?

Reply:
Yes, they're for us to reference if we ever need to swap back to secure mode (like if that's what the re-audit recommends).